src/views/MyProfile/Components/Identification/defaultGordonImage.js

```diff
@@ -0,0 +1,5 @@
R9zf5xugipOaStDCO98FoRFSooIuJnpYtaBMkM9G+7U6bc7bnbvjOj+eD3ndRhFSNaJKjRCCJI5H526gNnNx+U4fHLbCNRMYKloDe1K4ggqcF0WCcQ42Xt15VE5hE3eI0k1kaCCJIXyNedYiCb2CyPzRSrQgdZLMf4WhBBCiZtx9Dg0zd0y+y4pHf9Nv9ZBuccNOxHEEEywczKzdXCPpqusjouT6PjPt66WQWdjlpWiSAFx4STowQooJHRKpbwMY0tKVarcR3rBREkEQbMqKzut8jKhQN39yC7z6RBDztGCPlGMQgiSKYoHzW/+iLMah7fiLlVmZKDp1EaCyNBBEkMVLBeP9y0lfMsx6K1GzEvtnE20n28vFARI5TEL4yJuji/64IXQWLhMzd2q+62Y1mUps0Ug4e1Hlf3NOF2VMmHT13f1fadkDkLIijvs9PYhdaFXRe+CBK52u2etYEkPIkbPFLzqgzJwUY8yuuSycIzK6yN1q9z0E8SL4ggvsC1w9uQFHXNT5ceraxJZhLik0Utw29ZFUVDhiEdLitJgrkjSZq/RJD6+AGcU2YNqt3B8ByEhmSksPnHZZ0LChEj5Pn9/SOsNCXqJKcg5xDvdCxGonE+Z13X1Y4W8FMfs/Ta+cJLqZ9SVPpFABEkEpC1Y5np126LdQ7DYRirhu9vPsOEqIHnlnsIKBOfnDQu8wZ5bboGsfGEpBheJx0OKnm8CJIaPCWw//db9uQzWEsSLMsY+8x8QTbQ7GM4D6/3FR2NPDEK0VfSqI5PfxJBTgOY6oRBA62yQRY5jjyFOkoaY2lnmePI7L/fpG9RFMBOcO76zkWQxIJDTBFQyyKLZxx02+EVdg/A5Cru/vyMn/1O1MD/1+njoB123fa9RfXcJaOl71oEKRjc89STBu7vHAujtyyLjFFixlOb9T2KIIVDtwmLM5GtOyMVwgwkChGke/U9iiAFw+QlGzIzg6NoSVYpixRu1GCYj75HEaRg4MLOKpyRa00y8vKNEmjR9D2KIAUD9YqsgicHTxC0UllqutjX+ElRyJZl2UUpggg1cG/Gtj6ICTnuNR3GRvp5DLcRKFLVR7FLzwjnxOBQCpH47t7UZaIdPe0Nlofy5BVBCgqarI5l2FqLBzAjCBr4eOm6A/8u0s44zgdpqhAkUn2fX7Xd9xgyexBBiqLjyrq9tsu4RfbY3Pn9AkkMg3HCzguShS3TGKMg2bsIUhSQos1ydiCjlLn7I7tn6YQGbF7VNmtAzRMhyjm1GjIn9DWuLh+r704EKR5+3LR/pibVN5zMaDkK3jjncmaecQ64PTJQR9+bCFJUMD8kKy8s+tqTnse5nv4WdyB4jPoUEkSQzEG/CI1MaWPjrn2JDbSHzl5dEOKJIEJmYE5HGusffvcrf3wo9usieEQe7xcM2nEkLYIIctpxVfmYxClgfs89pTYqmFcYFLQG63sRQUqikHg8AUFIOWNdFBQoj/WdiCC1rpU334DQIIseLInivBbj3kgRh/V+0DWp70UEqVVIMgaa6nfc9C7GcPnaf/Eilv5KBKlVvlsMAY0bSWxQMXxgtHMUUSSaLYwg9P2IIKcNF7YcZhuSqIjHmVWIc0lSGUiQPMUvaO+FKLim6PsSQYoG+tjxw3VcSPC/wi8Xa1DGLOSb+5H2gm0zrCLW0+rZvQet/uuHd/fLbHipCCLUAEspDOj8XBaRpa/YuCvUgbHnpKWZLXt4MsRNMUNoEgojK6pM8/4zrdoXdQC6sPosTRFBUuKXzQfazsKkNQ/EiNifZn1eLLfSTrmi2Qp7I55sg2autD0nnOvHGnWuN70kIkjKinkSJxLSuNNXbLKq2rdeUDjZOcs6Ap8vdwp44qL19v+mLd9kN+1xyc2TkPP3s0cVQQSLnzbrH+nCQjZC/wV9HEzVpQ8cc7pi3IFpvIIMLJecgaUEc9WduSMsoTCsw9ExLAvGXoWZjiwX3cF+q5TrKyJIwkYp7r75gp9hRMHpvIAoIH7xlp62icrxAXYar7znRY0E/2G/IMHAZh61MMvC+mKELYIkABdbvtED9IcgHqxN582gndt7TjGNe021PSdU+r0/c1Hr4aHvCzkLWTmeKG4b1lKtq4ggCXB+2dC8Tw/W6HX1/fV6fHn1+8CjmPfiznYxievStqOq/61izTZ70xBBBIvrO4236/Xhc9YEuq4PmbW6zr4/9i74dJG1OjGNqszuY5h9+I3cktGZKc/TBtUyxhENGmqJJZzEdxv3sdonBt4EBXdhfVYiSL3OYoUFnYX6nESQeov+0ytDCVKXl1iCCJIKSNGpbeSbQajPSgSpl0Btu/VkUY0//QqGGLedKfGfCFJf3UtwHsH+k86/PftrGscxvi2J8YIggtR5oJ8i3en8PWhOCPINfV4iSL3Hrd0m+RKE0c7qAxdB6j0+cW2XwPHRyDr0GYkg9R7M5PALNFsaNSCC1Htc3HpEqM8VAzo/rP5vEaS+gpRuvjnnO/YetAYOcT2vBBGkJEAveJQ4eOiIbdO9p89087NmA8wHLivX5yeClD5+1LRfot7v7c8fsPMJ8bbS5yiClCwYpBMWNBbRlRc0PhpzhA9e3kGfpQhSmvjBXX3zuoTQ131280Hmth5TfHtJ8LXSZymClCTYS7hbUcP2IMzqaNJrao3/q9y0O7YvryCC1Blc2X5MtUEceqxlG3ZaQngDyyB+vs/UFdX/9szOfWZUxVpV30WQ0ga2Pr9uMdiKFXka0MON5y5GDvhTbd6933z7jhPzP7DfuabDWGtYTSsr1qMN9AQRQQRBBBEEEUQQRBBBEEEEQRBBBEEEEQQRRBBEEEEQQQRBBBEEEUQQRBBBEEEEQQTRhyAIIoggiCCCIIIIgggiCCKIIIgggiCCCIIIIggiiCAIIoggiCCCIIIIgggiCCKIIIgggiCCCIIIIggiiCCIIPogBEEEEQQRRBBEEEEQQQRBBBEEEUQQRBBBKEX8C6TKJBu4JKZpAAAAAElFTkSuQmCC';
```

Comment:
is this auto generated? if not I'm confused as to where this comes from?

Reply:
If you reset your profile image, the server will return your default profile picture. In Development, it's a picture of the lion on its hind legs with the Gordon blue background. In Production, it will be the same picture on your ID. I just used that string (which I got by going looking at the DOM elements and taking it from the src attribute in the image tag) as the fallback if the user resets their picture and it fails to retrieve their default from the database. I just have it show that Gordon picture so the user sees that their profile picture has changed. 

Before I added that, if the user reset (same as deleting) their preferred picture and the fetch for their default picture failed afterward, their preferred picture would still show. That would make the user confused because they would've received the notification from the Snackbar that they've successfully reset their picture. Since I can't just show an empty picture, I needed a fallback picture that I can use to temporarily display. Once the user refreshes the browser or leaves the page, that Gordon picture is no longer shown.

In other words, I have the picture there just in case if the fetching the user's profile image fails. It's a fallback that can help prevent the Identification Card from being stuck in the loading phase